script.js

```diff
@@ -1,12 +1,395 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+// ---------------------------------Global Variables------------------------
+
+let allEpisodes;
+const allShows = getAllShows();
+
+// root element
+const rootElem = document.getElementById("root");
+
+//creating the parent div for jumbotron
+const jumbotron = document.createElement("div");
+
+//creating the go back button 
+  const goBackBtn = document.createElement("p");
+  
+//create input & select and the parent div
+let searchDiv = document.createElement("div");
+searchDiv.classList.add("searchDiv");
+const searchShow = document.createElement("select");
+const searchByEpisode = document.createElement("select");
+const search = document.createElement("input");
+
+//create parent div for card episodes
+let episodesDiv = document.createElement("div");
+
+//create parent div for shows cards
+let showsDiv = document.createElement("div");
+
+//create number of episodes paragraph
+let numberOfEp = document.createElement("p");
+
+rootElem.appendChild(jumbotron);
+rootElem.appendChild(goBackBtn);
+rootElem.appendChild(searchDiv);
+rootElem.appendChild(numberOfEp);
+rootElem.appendChild(episodesDiv);
+rootElem.appendChild(showsDiv);
+
+
+//display shows' page
+function setupShowsPage() {
+  makeJumbotron();
+  makeShowSelector();
+  makeShowSearchShow();
+  createAllShows(allShows);
+  makeFooter();
+}
+
+//creating the jumbotron
+function makeJumbotron() {
+  jumbotron.classList.add("jumbotron");
+
+  const mainHeading = document.createElement("h1");
+  mainHeading.innerHTML = "Unlimited Movies, TV Shows and More";
+  mainHeading.classList.add("mainHeading");
+  jumbotron.appendChild(mainHeading);
+}
+
+//creating the go back button 
+function makeGoBackBtn (){
+  goBackBtn.innerHTML = "☚ Go back ";
+  goBackBtn.setAttribute("id", "goBackBtn");
+
+  //add eventListener on click for goBackBtn
+  function handleBackBtn() {
+    window.location.reload();
+  }
+
+  goBackBtn.addEventListener("click", handleBackBtn);
+}
+
+
+//creating the shows' select input
+function makeShowSelector() {
+  searchShow.classList.add("searchInput");
+  searchDiv.appendChild(searchShow);
+  searchShow.placeholder = "All shows";
+
+  const initialOptionShow = document.createElement("option");
+  initialOptionShow.value = "";
+  initialOptionShow.innerHTML = "Select a show";
+  searchShow.appendChild(initialOptionShow);
+
+  allShows.forEach((show) => {
```

Comment:
> vi. This show select must list shows in alphabetical order, case-insensitive.

This is from requirement 2 for level 400.
Your show selector must be in alphabetical order, therefore you don't met level 400